routes.py

```diff
@@ -1,13 +1,36 @@
-from flask import Flask, request, Response, jsonify
+from flask import Flask, request, Response, jsonify, redirect, url_for, session
 from flask_cors import CORS, cross_origin
 from flask import Flask, make_response
 from scraper.scraper import get_profile, scrape
+from werkzeug.utils import secure_filename
 import json
+import os
+
 app = Flask(__name__)
 
+UPLOAD_FOLDER = '/Users/samurai/code/couleur/be_flask/'
 cors = CORS(app)
 app.config['CORS_HEADERS'] = 'Content-Type'
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+app.secret_key = "70l0"
```

Comment:
if it's a secret then it should be in some config file (not pushed on github). crawlers look for these keywords